### PR TITLE
Add ESM importmap to features list

### PR DESCRIPTION
**Pull request by Fedik**
Add ESM importmap to features list

### CODE REVIEW

**Comment by HLeithner** [comment]
Link is broken?

**Comment by Fedik** [comment]
It seems, but I not very see why, for my it is correct

File: migrations/44-50/new-features.md [hunk 1]
```diff
@@ -18,3 +18,10 @@ File: libraries/src/MVC/Model/State.php
 Description: The state of a model is not anymore a `CMSObject`. Instead of, it is a new class libraries/src/MVC/Model/State.php which inherits from a `Registry`. Like that are all functions available a state registry should have. The class itself is deprecated as only features from the registry should be used. In version 7.0 we are going to remove the state class all together and use then only the registry class.
 
 For backwards compatibility, the state class supports direct property access though magic methods which will throw a deprecated warning when used.
+
+
+#### Support for ESM importmap
+
+Support of [ESM importmap](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap) 
+provides a possibility to keep the code modular, flexible and allows to use existing ES modules with minimum recompilation. 
+For more detail about how to register your module in to `importmap` look for [Working with ESM importmap](/docs/using-core-functions/web-asset-manager.md#working-with-esm-importmap) in Web Asset Manager.
```
Comment (HLeithner) [inline migrations/44-50/new-features.md:27]:
```suggestion
For more detail about how to register your module in to `importmap` look for [Working with ESM importmap](/docs/using-core-functions/web-asset-manager#working-with-esm-importmap) in Web Asset Manager.
```

**Comment by HLeithner** [comment]
I removed the .md postfix, maybe it fixes it.

**Pull request merged**